presto-docs/src/main/sphinx/functions/datetime.rst

```diff
@@ -57,6 +57,10 @@ Date and Time Functions
 
     This is an alias for ``CAST(x AS date)``.
 
+.. function:: last_day(x) -> date
```

Comment:
Discussed with @martint and we should call this `last_day_of_month` so it's not ambiguous (with last day of year).